pom.xml

```diff
@@ -54,6 +60,16 @@
         <tag>HEAD</tag>
         <url>https://github.com/jenkinsci/archetypes/</url>
     </scm>
+    <distributionManagement>
```

Comment:
Did you intend for this to be merged with #646?

Reply:
Partly, yes